src/rviz/robot/robot.h

```diff
@@ -39,8 +39,7 @@
 #include <OgreQuaternion.h>
 #include <OgreAny.h>
 
-#include <urdf_model/types.h>
-#include <urdf_world/types.h>
+#include <urdf/model.h> // can be replaced later by urdf_model/types.h
```

Comment:
This can be changed to include urdf_model/types.h as soon as https://github.com/ros/urdfdom_headers/pull/33 is merged.